### PR TITLE
chore(ui-react-notifications): add package boilerplate

### PR DESCRIPTION
**Pull request by sreeramsama**
#### Description of changes
Add package boilerplate

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-core-notifications/.estlintrc.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  extends: ['amplify-ui/react'],
```
Comment (calebpollman) [inline packages/react-core-notifications/.estlintrc.js:2]: Looks like this need to be updated to use the scoped namespace (same with line 8)

File: packages/react-notifications/src/__tests__/index.spec.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import * as authExports from '..';
```
Comment (calebpollman) [inline packages/react-notifications/src/__tests__/index.spec.ts:1]: `authExports`?

File: packages/react-notifications/src/version.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const VERSION = '0.0.1';
```
Comment (calebpollman) [inline packages/react-notifications/src/version.ts:1]: Don't think we need this?

File: packages/react-core-notifications/.estlintrc.js (hunk 1 above)
Comment (sreeramsama) [reply packages/react-core-notifications/.estlintrc.js:2]: Oh yeah
Updated in the later PRs 
Will fix here too

**Review by calebpollman — APPROVED** [review]

**Review by 0618 — APPROVED** [review]

**Pull request merged**